app/src/home.js

```diff
@@ -567,22 +577,43 @@ export default function Home() {
     stage = STAGE_HOME;
   } else if (audioStream && !showAnalysis) {
     stage = STAGE_TRANSCRIBING;
-  } else if (!showAnalysis) {
+  } else if (showAnalysis) {
     stage = STAGE_TRANSCRIBED;
   } else if (!showExport) {
     stage = STAGE_SUMMARIZE;
   } else {
     stage = STAGE_EXPORT;
   }
 
+  const onComprehendResultDelete = (r) => {
+    setComprehendResults((prevResults) =>
+      prevResults.map((prevResult) => {
+        const a = prevResult.findIndex((result) => result.id === r.id);
+
+        if (a === -1) return prevResult;
+
+        return [...prevResult.slice(0, a), ...prevResult.slice(a + 1)];
+      }),
+    );
+  };
+
+  const onComprehendResultAddition = (val, category) => {
```

Comment:
I would expect the addition to show as the first item in the list, right below the input element. Let's see how others feel.

Reply:
Updated